zh:
    "InternalError": "异常（覆盖）%s" # 覆盖默认的翻译